src/jansson/CHANGES

```diff
@@ -1,3 +1,783 @@
+Version 2.14
```

Comment:
I see this is Jansson 2.14, not 2.1.4 as the PR title and the title of commit f5624cf975223a326eed45d26d909df766c55028 seem to indicate.